README.md

```diff
@@ -271,6 +271,36 @@ Add to your repository as `.github/workflows/sile.yaml`.
 This work flow assumes your project has a source file `my-document.sil` and will leave behind a `my-document.pdf`.
 Note the comments in [the section about Docker](#docker) regarding version tags.
 
+## Installing third-party packages
+
+Third-party SILE packages rely on the `luarocks` package manager.
+
+To install the latest version, for the given package, you may often use its provided “rockspec”:
+
+```bash
+# This example uses markdown.sile, a highly useful plugin that converts Markdown documents to SILE `.sil`.
+luarocks install markdown.sile --local
+```
+
+You may remove `--local`, in which case a global installation will be done (on most systems this requires `root`).
```

Comment:
> on most systems this requires `root`
Or adding `sudo`? (keeping in mind Linux users are not always devs and that terminal-friendly)

Reply:
I don't mind hints reminding people to use their system package manager, but (with by distro developer hat on here) I'm against doing so in anything other than the default options and minimal hints. There are many ways to customize most install commands as a downstream project we don't want to wade into that too much. There are also many ways of running root. Some package managers handle achieving root themselves, others expect you to run them in a root shell, with `sudo`, with `doas`, or some other tooling. People should follow their distro usage guidelines for which way to achieve elevated permissions, we don't want to spec that.